Source/connector/Connector.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+namespace WPEFramework {
+
+namespace PluginHost {
+    struct IShell;
+}
+
+namespace Connector {
```

Comment:
Although technically there is nothing wrong with this, I would like to hide it from the external interface and force all users to the C functions, to make sure that there is only one way for accessing for these methods. Less exposure is less maintenance and less portability risk. 

So lets hide this part: https://github.com/rdkcentral/Thunder/pull/891/files#diff-276434645d979f1b866322ebbce0d324c54b0a94e89821f4f50afe7d77107b1cR28-R51 and only allow the C functions to be accessed. This class off cource can be sused interbnally to this implementation thats fine, so it is just to move the class to the cpp file...

Reply:
Done